From 32421a72443c4d42efc3fc627ec1874e0a833e3c Mon Sep 17 00:00:00 2001
From: Stefan Chulski <stefanc@marvell.com>
Date: Wed, 7 Feb 2018 14:44:02 +0200
Subject: [PATCH 2234/2241] net: mvpp2x: change default queue mode from single
 to multi

Default were changed to keep MUSDK API compatibility.

Change-Id: I6a0f4e756e86aa04c8e4aed1684aed4d99f11b71
Signed-off-by: Stefan Chulski <stefanc@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/50172
Reviewed-by: Yuval Caduri <cyuval@marvell.com>
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Shlomi Gridish <sgridish@marvell.com>
Reviewed-by: Omri Itach <omrii@marvell.com>
---
 drivers/net/ethernet/marvell/mvpp2x/mv_pp2x_main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/marvell/mvpp2x/mv_pp2x_main.c b/drivers/net/ethernet/marvell/mvpp2x/mv_pp2x_main.c
index 6bac51b..71fd991 100644
--- a/drivers/net/ethernet/marvell/mvpp2x/mv_pp2x_main.c
+++ b/drivers/net/ethernet/marvell/mvpp2x/mv_pp2x_main.c
@@ -88,7 +88,7 @@ u8 mv_pp2x_num_cos_queues = 4;
 
 u8 mv_pp2x_rx_count = MVPP2_DEFAULT_RX_COUNT;
 
-static u8 mv_pp2x_queue_mode = MVPP2_QDIST_SINGLE_MODE;
+static u8 mv_pp2x_queue_mode = MVPP2_QDIST_MULTI_MODE;
 static u8 mv_pp2x_used_addr_spaces;
 static u8 rss_mode = MVPP2_RSS_5T; /* Set 5-tuple default RSS mode */
 static u8 default_cpu;
-- 
2.7.4

